tests/network/network.go

```diff
@@ -202,6 +363,37 @@ func (n *LocalNetwork) GetPrimaryNetworkInfo() interfaces.SubnetTestInfo {
 	}
 }
 
+func (n *LocalNetwork) GetSubnetInfo(subnetID ids.ID) interfaces.SubnetTestInfo {
+	for _, subnet := range n.Network.Subnets {
```

Comment:
There are a handful of places where we do a linear search over the subnets and nodes. I didn't think it was worth migrating to maps for random access, given that we're only ever searching over a handful of subnets/nodes.